quiz.js

```diff
@@ -10,66 +9,53 @@
 
 // The character to use should be from user input in a <input type="text"> field in the DOM.
 
+// It accepts a single object as an argument. The object should have two key/value pairs.
 
 var tree = {
 	height: userHeight,
 	character: userChar
 };
 
-
-// function growTree(height, char) {
-// 	// body... call variables inside so they don't come up as undefined
-// }
-
-
-
 var treeButton = document.getElementById("yourTree");
 
+//This function is for the Enter button
 
+function enterInput(event) {
+	if (event.keyCode === 13) {
+		checkInput(true);
+	}
+};
 
+// If either of the input fields does not have a value in it when the user presses the enter key, or presses the button, then display an alert stating that both fields must have a value.
 
 function checkInput(yes) {
```

Comment:
For some reason I thought I needed to pass a value into the function, but I see I didn't use it anywhere inside, so I removed it.